backend/protocol_rpc/transactions_parser.py

```diff
@@ -1,21 +1,55 @@
 # rpc/transaction_utils.py
 
 import rlp
-import codecs
+from rlp.sedes import text
+from rlp.exceptions import DeserializationError, SerializationError
+from eth_account import Account
 from eth_account._utils.legacy_transactions import Transaction, vrs_from
 from eth_account._utils.signing import hash_of_signed_transaction
-from eth_account import Account
 from eth_utils import to_checksum_address
 from hexbytes import HexBytes
-from rlp.sedes import binary
+
 from backend.protocol_rpc.types import (
-    DecodedTransaction,
-    DecodedMethodCallData,
     DecodedDeploymentData,
+    DecodedMethodCallData,
+    DecodedTransaction,
 )
 
 
-def decode_signed_transaction(raw_transaction) -> DecodedTransaction:
+class Boolean:
+    """A sedes for booleans
+    Copied from rlp/sedes/boolean.py
+    Adding custom logic to also handle `False` as `0x00`, since the Frontend library sends `False` as `0x00`
+    """
+
+    def serialize(self, obj):
+        if not isinstance(obj, bool):
+            raise SerializationError("Can only serialize integers", obj)
```

Comment:
or booleans?

Reply:
This was all copy pasted from rlp/sedes/boolean.py, with a slight modification for handling `b"\x00"`
- I think it's best to not modify it a lot so that we don't diverge a lot from the original implementation
- `serialize` is not being used at the moment actually

Reply:
I meant the Error message :p